content/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx

```diff
@@ -4228,6 +4250,53 @@ Informs about those event properties that were deleted.
 | ------------ | ------------------------------------------------------------------------------------------------------------ |
 | `properties` | This is not a full `properties` object. This push shows only the properties that have been recently updated. |
 
+## Threads
+
+### `thread_deleted`
+
+Informs that a thread was deleted.
+
+<CodeResponse title={'Sample push payload'}>
+
+```json
+{
+  "chat_id": "PJ0MRSHTDG",
+  "thread_id": "K600PKZON8"
+}
+```
+
+</CodeResponse>
+
+#### Specifics
+
+|                        |                     |
+| ---------------------- | --------------------|
+| **Action**             | `thread_deleted`    |                                                   |
+| **Webhook equivalent** | -                   |
+
+### `threads_deleted`
+
+Informs that a threads were deleted.
```

Comment:
```suggestion
Informs that several threads were deleted. Threads could have been selected based on a date range and/or a tag.
```

Reply:
Is the info I added correct? It's something I assumed based on the push payload.

Reply:
I have just commited suggestion from @litwinow in this exact place : "Informs that threads from specific time window were deleted."

Reply:
it's correct - I will change it (but is not 'and/or' just 'or')

Reply:
Alternative description: 

> Informs that several threads from a specific date range or with the same tag were deleted.